docs/en/03_Upgrading/index.md

```diff
@@ -1022,14 +1022,14 @@ If you are using a modified `index.php`, `.htaccess`, or `web.config`, you will
   * `assets`
   * Any `favicon` files
   * Other common files that should be accssible in your project webroot (example: `robots.txt`)
-* Delete the root `resources` directory if present.
+* Delete the root `resources` or `_resources` directories if present.
```

Comment:
Worth mentioning here something like "or whatever your custom `resources-dir` value is in `composer.json`?

Reply:
This would only apply to someone using using S4.4 or greater without the public web-root who decided to update the public webroot. I think bringing in the _resources-dir value in composer.json_ here would make thing more confusing for users.